app.go

```diff
@@ -45,15 +46,15 @@ func setupServer() *mux.Router {
 }
 
 func main() {
-
+	fmt.Println("Good Morning, Sir")
 	flag.Parse()
 
 	r := setupServer()
 
 	fmt.Printf("Starting server on port:%d", port)
-	http.ListenAndServe(fmt.Sprintf(":%d", port), r)
+	log.Fatal(http.ListenAndServe(fmt.Sprintf(":%d", port), r))
 }
 
 func signHandler(rw http.ResponseWriter, r *http.Request) {
```

Comment:
this doesnt seem to be called anywhere. I'm guessing you wanna add this route / endpoint to the server? if so, i think we might be able to call `controllers.SignupHandler` directly in the `setupServer` function